includes/misc-functions.php

```diff
@@ -1971,12 +1977,14 @@ function give_goal_progress_stats( $form ) {
 			$actual = apply_filters( 'give_goal_donors_target_output', give_get_form_donor_count( $form->ID ), $form->ID, $form );
 			break;
 		default:
-			/**
-			 * Filter the form income.
-			 *
-			 * @since 1.8.8
-			 */
-			$actual = apply_filters( 'give_goal_amount_raised_output', (new DonationQuery())->form($form->ID)->sumIntendedAmount(), $form->ID, $form );
+            /**
+             * Filter the form income.
+             *
+             * @unreleased Revert changes implemented on the 3.14.0 version and add placeholder condition
+             * @since 3.14.0 Replace "$form->earnings" with (new DonationQuery())->form($form->ID)->sumIntendedAmount()
+             * @since 1.8.8
+             */
+            $actual = $usePlaceholder ? 0 : apply_filters( 'give_goal_amount_raised_output', $form->earnings, $form->ID, $form );
```

Comment:
Now that you have simplified this file, I don't think we even need the `$usePlaceholder` conditionals and filter.  We can hook into the existing filters as they are doing something really simple.  We could make a little helper to be reused for using these filters, but the advantage would be that we are not modifying this function anymore.  Exampe:

```php
if (apply_filters('givewp_admin_list_form_view_goal_progress_stats_use_placeholder', false)){

  add_filter("give_goal_amount_raised_output", function($output) {
      return 0;
  });
  
  add_filter('give_goal_progress_stats', function($stats) {
      $stats['actual'] = AsyncDataHelpers::getSkeletonPlaceholder('1rem');
  
      return $stats;
  });
}
```

Reply:
@jonwaldstein I think it will not work because passing 0 to the `give_goal_amount_raised_output` filter will not prevent other functions associated with this filter get triggered (including the one that calc the output retrieving real data from DB and slows down the page load) and we are trying to prevent exactly that. 

So, what the current logic does is prevent the `give_goal_amount_raised_output` filter from being triggered when using the placeholder and we need to do that to avoid triggering the function that slows the page load.

Reply:
Ok, I think I found a way to make it work and was able to fully remove the filter to enable the placeholder. Now everything is enabled on the Service Provider. Resolved in 86784519beced7fd0ca1cd7d2248853091da4cd4 and 6a795e52fd1cc0a629c49b3a88701079debcf723 and 882da69f1cce0ad6562ab4d31aa409bd1ad2b85f and 687caa50e31ba3b9b0f60ead1b491bb6e1c713a7 and 567f8d0f9306da2fb6015ddc159ad91947affe26